contracts/hydro/src/contract.rs

```diff
@@ -1295,6 +1328,129 @@ fn withdraw_icq_funds(
         }))
 }
 
+// This function will add a given liquidity deployment to the deployments that were performed.
+// This will not actually perform any movement of funds; it is assumed to be called when
+// a trusted party, e.g. a multisig, has performed some deployment,
+// and the contract should be updated to reflect this.
+// This will return an error if:
+// * the given round has not started yet
+// * the given tranche does not exist
+// * the given proposal does not exist
+// * there already is a deployment for the given round, tranche, and proposal
+// Having more arguments rather than nested structures actually makes the json easier to construct when calling this from the outside,
```

Comment:
We could construct `LiquidityDeployment` from input parameters in the `execute()` function and pass it to `add_liquidity_deployment()`?

Reply:
nice idea, thanks!